tests/sentry/grouping/snapshots/test_grouphash_metadata/test_metadata_from_variants/newstyle@2023_01_11/unreal_assertion_check_fail_android.pysnap

```diff
@@ -1,13 +1,13 @@
 ---
-created: '2025-02-19T15:24:52.069343+00:00'
+created: '2025-02-19T18:18:54.886839+00:00'
 creator: sentry
 source: tests/sentry/grouping/test_grouphash_metadata.py
 ---
 hash_basis: stacktrace
 hashing_metadata: {
   "num_stacktraces": 1,
   "stacktrace_location": "exception",
-  "stacktrace_type": "system"
+  "stacktrace_type": "in_app"
```

Comment:
We now have two different hashes for the stack trace rather than just a system one.